edgelet/hsm-sys/azure-iot-hsm-c/tests/CMakeLists.txt

```diff
@@ -4,6 +4,16 @@
 include(../deps/c-shared/configs/azure_iot_build_rules.cmake)
 usePermissiveRulesForSamplesAndTests()
 
+enable_testing()
+set(save_ut ${run_unittests})
+set(run_unittests OFF CACHE BOOL "unittests" FORCE)
+add_subdirectory(../deps/c-shared/testtools/ctest c-shared/testtools/ctest)
```

Comment:
Question: This was done because these dont use the permissive rule?

Reply:
This was done because I needed ctest, testrunner and umock-c to have permissive rules (see line5), but I didn't want to call usePermissiveRulesForSamplesAndTests() at the top level.